PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeClient.java

```diff
@@ -13,33 +31,44 @@
 
     private final BraintreeClient braintreeClient;
     private final PayPalInternalClient internalPayPalClient;
+    private final TokenizationClient tokenizationClient;
+
     private PayPalClient payPalClient;
 
     public PayPalNativeClient(BraintreeClient braintreeClient) {
         this.braintreeClient = braintreeClient;
         this.internalPayPalClient = new PayPalInternalClient(braintreeClient);
+        this.tokenizationClient = new TokenizationClient(braintreeClient);
     }
 
-    public void tokenizePayPalAccount(final FragmentActivity activity, final PayPalRequest request, final PayPalNativeFlowStartedCallback callback) {
+    public void tokenizePayPalAccount(final FragmentActivity activity, final PayPalRequest request, final PayPalNativeFlowCallback callback) {
         if (request instanceof PayPalNativeCheckoutRequest) {
             sendNativeCheckoutRequest(activity, (PayPalNativeCheckoutRequest) request, callback);
         } else if (request instanceof PayPalNativeVaultRequest) {
             sendVaultRequest(activity, (PayPalNativeVaultRequest) request, callback);
         }
     }
 
-    private void sendNativeCheckoutRequest(final FragmentActivity activity, final PayPalNativeCheckoutRequest request, final PayPalNativeFlowStartedCallback callback) {
+    private void sendNativeCheckoutRequest(final FragmentActivity activity, final PayPalNativeCheckoutRequest request, final PayPalNativeFlowCallback callback) {
+        //This is copied from PayPal module.
+        braintreeClient.sendAnalyticsEvent("paypal.native.single-payment.selected");
+        if (request.getShouldOfferPayLater()) {
+            braintreeClient.sendAnalyticsEvent("paypal.native.single-payment.paylater.offered");
+        }
+
```

Comment:
Copied this from `PayPal` module, is `single-payment` the correct analytic?

Reply:
Yeah these analytics event strings haven't been updated to align with the new method naming, so I think it's ok to match them to the events in `PayPalClient` for now.